### DISCUSSION TITLE
Export Manage Object/JSON Tree data type

### DISCUSSION THREAD

**Discussion by jcdevz-dev** [AntlerVC/firetable#1199] [Support / Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I'm exporting data. I don't have much freedom on object/json data types on how would I like them to be formatted and its row name. For instance that, I just want to export the id of the artist object. example -> custom_field: artist.id without worrying about table structure or adding functions just for exporting a custom format of the data. 

**Describe the solution you'd like**
It would be better if I click the export button, like always it will show the fields. But if an object/json is present, it will become a dropdown, like a folder structure and I will be able to select specific field of the object. Also I would like it if I can add a new field and map the field that I want.

**Describe alternatives you've considered**

**Additional context**

**Comment by harinij** [comment, marked as answer]
Hi @jcdevz-dev 👋 this is possible in Rowy by spreading the JSON object easily into its own columns, this will not create any additional function or create additional datapoint. I have created this quick video to explain how. Let us know if this helps your usecase. Thanks.

https://user-images.githubusercontent.com/307298/229473649-9d916ddb-c4ed-40a1-83ca-b49486b5c247.mp4

**Reply by jcdevz-dev** [reply]
Hello @harinij  thank you for your response. I see that this is possible and will I definitely use this. Many thanks!
